src/app/popup/popup.html

```diff
@@ -14,12 +14,18 @@
       <div class="text-section-header">Update Scanner</div>
     </div>
 
+    <div class="panel-section panel-section-queue-state hidden" id="scan-state">
```

Comment:
I think this might be better just above the panel-section-footer,  so that the list items don't change absolute position when scanning starts and stops.

Reply:
Wouldn't that cause basically the same problem? Maybe above all updates would solve this?

Reply:
The problem is when the div is shown/hidden, the list items move down and up under the user's mouse pointer. Having the div after the list means the items don't move when the div is shown/hidden (the list just changes size).

It's a very minor change, I can just do it after merging.

Reply:
Yes I understand the problem, but I read wrong for some reason I thought you meant header not footer.